apps/dashboard/app/views/layouts/application.html.erb

```diff
@@ -48,19 +44,19 @@
           <% else %>
             <%= render partial: 'layouts/nav/custom_navigation', locals: { navigation: @nav_bar }%>
           <% end %>
-        </ul>
-
-        <ul class="navbar-nav">
-          <% if @help_bar.empty? %>
-            <%= render partial: 'layouts/nav/develop_dropdown' %>
-            <%= render partial: 'layouts/nav/help_dropdown' %>
-            <%= render partial: 'layouts/nav/user' %>
-            <%= render partial: 'layouts/nav/log_out' %>
-          <% else %>
-            <%= render partial: 'layouts/nav/custom_navigation', locals: { navigation: @help_bar, menu_alignment: 'dropdown-menu-right' }%>
-          <% end %>
-        </ul>
-      </div>
+
+          <div class="ml-auto d-flex">
```

Comment:
Regarding the navigation look and feel, If we change from:
` <div class="ml-auto d-flex">`
to
` <div class="ml-auto navbar-collapse">`

will vertically center the navigation items for the help bar, making the look and feel more consistent when there is no space

Reply:
nevermind, the helpbar is left aligned with the navbar-collpase. Looking into alternatives

Reply:
This `div` is definitely the cause of the issues or at least the biggest cause. 

There may be a way to figure out which element is where and apply an `ml-auto` (or `mr-auto`) to the appropriate element dynamically somewhere.

Reply:
This seem to do the trick:
`<div class="ml-auto d-flex align-items-center">`

Reply:
Yea I can add `align-items-center` to the outter most `ul` - I think what I really need is an invisible div that adds the space between the `nav_bar` and `help_menu` instead of wrapping the `help_menu` inside a div.  That'll likely help me fix the tests too when they all share the same general structure.

Reply:
May have fixed it, it seems like I tried this before, but it didn't work for whatever reason then...

In any case, just adding this div between the two seems to do the trick (instead of wrapping the help_menu in a parent div)
`<div class="ml-auto"></div>`

Reply:
Changes look good locally 👍

Reply:
Thanks for the help!